isis/appdata/serialnumbers/Viking1CameraASerialNumber.trn

```diff
@@ -0,0 +1,33 @@
+Group = Keyword1
+  Auto
+  InputKey       = SpacecraftName
+  InputGroup     = "IsisCube,Instrument"
+  InputPosition  = (IsisCube, Instrument)
+  OutputName     = Keyword1
+  OutputPosition = (Group, SerialNumberKeywords)
+  Translation    = (Viking2, VIKING_ORBITER_2)
+  Translation    = (*, *)
+End_Group
+
+Group = Keyword2
+  Auto
+  InputKey       = InstrumentId
+  InputGroup     = "IsisCube,Instrument"
+  InputPosition  = (IsisCube, Instrument)
+  OutputName     = Keyword2
+  OutputPosition = (Group, SerialNumberKeywords)
+  Translation    = (VISB, VISUAL_IMAGING_SUBSYSTEM_CAMERA_B)
```

Comment:
Does "B" need to be in here if the file is just for "A" ?

Reply:
Since I now have a file for each instrument, no they don't I'll update